### DISCUSSION TITLE
Failed to export metrics. Server responded with UNIMPLEMENTED. This usually means that your collector is not configured with an otlp receiver in the "pipelines" section of the configuration. If export is not desired and you are using OpenTelemetry autoconfiguration or the javaagent, disable export by setting OTEL_METRICS_EXPORTER=none. Full error message: unknown service opentelemetry.proto.collector.metrics.v1.MetricsService

### DISCUSSION THREAD

**Discussion by wesleyzup** [open-telemetry/opentelemetry-java-instrumentation#7905] [Q&A]
I'm using the spring boot autoconfigure with the following dependencies: 

```xml
    <dependencyManagement>
         <dependency>
                <groupId>io.opentelemetry.instrumentation</groupId>
                <artifactId>opentelemetry-instrumentation-bom-alpha</artifactId>
                <version>1.22.1-alpha</version>
                <type>pom</type>
                <scope>import</scope>
            </dependency>
        </dependencies>
    </dependencyManagement>
    
    <dependencies>
           <dependency>
            <groupId>io.opentelemetry.instrumentation</groupId>
            <artifactId>opentelemetry-spring-boot-starter</artifactId>
        </dependency>
         <dependency>
            <groupId>io.opentelemetry.instrumentation</groupId>
            <artifactId>opentelemetry-instrumentation-annotations</artifactId>
        </dependency>
     </dependencies>
```
SpringBoot autoconfiguration for opentelemetry
```java
@Configuration
@EnableOpenTelemetry
public class SpringBootAutoConfigure {

}
```
Jaeger container configuration
``` shell
docker run -d --name jaeger \
  -e COLLECTOR_OTLP_ENABLED=true \
  -p 6831:6831/udp \
  -p 6832:6832/udp \
  -p 5778:5778 \
  -p 16686:16686 \
  -p 4317:4317 \
  -p 4318:4318 \
  -p 14250:14250 \
  -p 14268:14268 \
  -p 14269:14269 \
  -p 9411:9411 \
  jaegertracing/all-in-one:1.42

```

The problem is that I can't disable the message:
**i.o.e.internal.grpc.OkHttpGrpcExporter   : Failed to export metrics. Server responded with UNIMPLEMENTED. This usually means that your collector is not configured with an otlp receiver in the "pipelines" section of the configuration. If export is not desired and you are using OpenTelemetry autoconfiguration or the javaagent, disable export by setting OTEL_METRICS_EXPORTER=none. Full error message: unknown service opentelemetry.proto.collector.metrics.v1.MetricsService**.

I already tried setting up the environment variable **OTEL_METRICS_EXPORTER=none**, but this variable isn't interpreted by opentelemetry-spring-boot-start, because the error continue being showed at the console log.

When I execute a curl to the jaeger container a received the error message **Received HTTP/0.9 when not allowed**.

The question are:
Is there a way to connect and send the metrics successfully to the jaeger ? 
What library should I use to fix this problem?
Is there some example to resolve this message using **opentelemetry-spring-boot-starter**  and jaeger? 

I'm waiting for your help.
Thank you

**Comment by trask** [comment]
hi @wesleyzup! it looks like this is configurable using `otel.exporter.otlp.metrics.enabled=false`:

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/53353acf03ecd4f1b7f19250ea339570ff2e75da/instrumentation/spring/spring-boot-autoconfigure/src/main/java/io/opentelemetry/instrumentation/spring/autoconfigure/exporters/otlp/OtlpExporterProperties.java#L29

**Reply by wesleyzup** [reply]
Hi @trask, thank you for you response.
What would I have to do to send the metrics to jaeger using OTLP and avoid the mentioned error above?
Because the default metrics endpoint is **localhost:4317**, but unfortunately the jaeger metrics connection isn't realized.
Is there some tips to help me ?

**Reply by laurit** [reply]
In your place I'd firstly try to figure out whether Jaeger supports metrics at all. If the answer is no then just disable sending metrics.
